src/libraries/System.IO.FileSystem.AccessControl/tests/FileSystemSecurityTests.cs

```diff
@@ -409,5 +416,85 @@ public void AuditRuleType_Returns_Valid_Object()
             Type type = fileSecurity.AuditRuleType;
             Assert.Equal(typeof(FileSystemAuditRule), type);
         }
+
+        [Fact]
+        public void DirNotFound_DirectorySecurity()
+        {
+            using TempDirectory tempDirectory = new TempDirectory();
+            {
+                string longDir = CreateLongDirectory(tempDirectory.Path, create: false);
+                Assert.Throws<DirectoryNotFoundException>(() =>
+                {
+                    var security = new DirectorySecurity(longDir, AccessControlSections.Owner);
+                });
+            }
+        }
+
+        [Fact]
+        public void FileNotFound_FileSecurity()
+        {
+            using TempDirectory tempDirectory = new TempDirectory();
```

Comment:
These could use using declarations
https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-8#using-declarations

Reply:
Actually you are already (by accident maybe? Was confused by the curlies)

Reply:
I should remove the curly brackets.